target/linux/layerscape/patches-4.9/304-dts-layerscape-add-traverse-ls1043.patch

```diff
@@ -0,0 +1,89 @@
+From c0612164b379ebc8964da6bc6f6ced9736dce488 Mon Sep 17 00:00:00 2001
```

Comment:
Where is this patch coming from?  Was it upstreamed?  What's its status if it was?

Reply:
This particular patch applies on top of NXPs patchset (in patches-4.9/), hence the number-sequence might be incorrect. Once layerscape is moved to 4.14, that should blow a lot of the uglyness away.

I haven't submitted our DTS to mainline yet, I think I should send the hwmon drivers up first so the bindings for those can be properly defined?